packages/squiggle-lang/src/js/parse.ts

```diff
@@ -1,23 +1,27 @@
-// import {
-//   errorValue,
-//   parse as parseRescript,
-// } from "../rescript/TypescriptInterface.gen";
-// import { result } from "./types";
-// import { AnyPeggyNode } from "../rescript/Reducer/Reducer_Peggy/helpers";
+import * as RSParse from "../rescript/Reducer/Reducer_Peggy/Reducer_Peggy_Parse.gen";
+import { result, resultMap2 } from "./types";
+import { AnyPeggyNode } from "../rescript/Reducer/Reducer_Peggy/helpers";
+import { SqLocation } from "./SqError";
 
-// export function parse(
```

Comment:
Do we need to keep all this commented code?

Reply:
This PR is a clean up of that commented code (which I commented out in 0.4).